projects/eslint-config/plugins/portal/lib/rules/no-metal-plugins.js

```diff
@@ -44,22 +54,42 @@ module.exports = {
 		},
 		fixable: null,
 		messages: {
+			'no-metal-affix':
+				'metal-affix is deprecated; use native offset instead',
 			'no-metal-ajax':
 				"metal-ajax is deprecated; use `import {fetch} from 'frontend-js-web';` instead",
+			'no-metal-anim':
+				'metal-anim is deprecated; use native `window.requestAnimationFrame()` instead',
```

Comment:
You could drop the "native" and the (implied) `window.` from here. We have some interest in keeping the ESLint error messages relatively narrow because they get spewed out to the console, or inlined in IDEs etc.

I'd say the same for all the ones that say "native" (I know there were already a couple of pre-existing ones but you should feel free to change those too).